pallets/parachain-staking/src/types.rs

```diff
@@ -219,8 +246,8 @@ where
 		}
 	}
 
-	// Returns Some(remaining balance), must be more than MinDelegatorStk
-	// Returns None if delegation not found
+	/// Returns Some(remaining stake for delecator), must be more than
```

Comment:
We should either use the infinite everywhere (e.g. Return) or the third person everywhere (e.g., Returns),

Reply:
I didn't find where we use only `return` and i think `returns` sounds better. But not sure... 😅 Added a dot in the end anyways. 2e35cbe8838996f08cbc4020913233440c965b2e

Reply:
Sorry I did not mean that there are other `Return`, but that there are other places that do not use the third person, like some `Check`, for instance.

Reply:
Ah i think i got it! Fixed here -> 90cfd09 <- maybe?